src/android/plugin/google/maps/PluginTotalPaveTileLayer.java

```diff
@@ -25,9 +39,9 @@ public void reload(final JSONArray args, final CallbackContext callbackContext)
         });
     }
 
-    private void $clearTileCache(String id, final CallbackContext callbackContext) {
+    private void $clearTileCache(TileOverlay overlay, final CallbackContext callbackContext) {
```

Comment:
pass in object reference instead of string key to hold reference instead of trying to wait for this code to run properly on the ui thread.